### PR TITLE
Add edit link url

### PR DESCRIPTION
**Pull request by jakobw**

### CODE REVIEW

File: src/store/RootState.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default interface RootState {
```
Comment (jakobw) [inline src/store/RootState.ts]: Will move this into a links module, as discussed.

File: tests/edge-to-edge/server/app.spec.ts [hunk 2]
```diff
@@ -202,11 +207,12 @@ describe( 'Termbox SSR', () => {
...
+		const editLink = '/some';
 
 		nockSuccessfulLanguageLoading( language );
 		nockSuccessfulEntityLoading( entityId );
 
-		request( app ).get( '/termbox' ).query( { entity: entityId, language } ).then( ( response ) => {
+		request( app ).get( '/termbox' ).query( { entity: entityId, language, editLink } ).then( ( response ) => {
 			expect( response.status ).toBe( HttpStatus.OK );
 
 			const $dom = getDomFromMarkup( response.text );
```
Comment (wiese) [inline tests/edge-to-edge/server/app.spec.ts]: Should we not assert that the link appears in the mark-up?

File: src/components/TermBox.vue [hunk 3]
```diff
@@ -14,7 +14,11 @@
 				<p class="wikibase-termbox__aliases wikibase-termbox__aliases--placeholder" v-else>?</p>
 			</div>
 		</div>
-		<div class="wikibase-termbox__action"></div>
+		<div class="wikibase-termbox__actions">
+			<div class="wikibase-termbox__action--edit">
+				<a :href="editLinkUrl">PEN</a>
```
Comment (wiese) [inline src/components/TermBox.vue]: This is not tested for in the component unit test.

File: tests/edge-to-edge/server/app.spec.ts (hunk 2 above)
Comment (jakobw) [reply tests/edge-to-edge/server/app.spec.ts]: yes! done

**Comment by jakobw** [comment]
> How do we sync this with the pen CSS?

This adds an invisible link so it could be merged as is?

**Pull request merged**